import os
import pandas as pd

try:
    token_auth = dmda.DomoTokenAuth(
        domo_instance="domo-dojo", domo_access_token=os.environ["DOMO_DOJO_ACCESS_TOKEN"]
    )

    await get_dataset_by_id(dataset_id=123, auth=token_auth)

except DatasetNotFoundError  